src/tests/ftest/datamover/copy_procs.yaml

```diff
@@ -0,0 +1,33 @@
+hosts:
+    test_servers:
+        - server-A
+    test_clients:
+        - server-B
+timeout: 600
```

Comment:
Looking at the PR-3802 build 10 results test took 6 mints 37 sec. Can we reduce timeout value to something like 8 mints.

Reply:
Actually, for build 10, that 6m 37s comes from both the CentOS and Leap15 runs, which are separate. Each one took a little under 3-4 minutes.
This is with multi processes on a VM, so timing isn't exact, but I think we can maybe bump this down to even 6 minutes and still be okay.
What do you think?